tests/tars/test_client.py

```diff
@@ -67,16 +65,18 @@ def setUp(self, mock_credential):
             resource="https://some_resource",
         )
         tars_virus_identifiers = TarsVirusIdentifiers(
-            virus_tars_id="AiV456",
-            plasmid_tars_alias="AiP123",
+            virus_tars_id="AiP123",
```

Comment:
I know this is fake example, but since virus_tars_id are usualy AiVXXX and plasmid_tars_aliases are usually AiPXXX it would be better to not have the virus_tars_id use AiP here to prevent confusion for someone reading it.